scripts/circleci-jobs/deploy_to_dev.sh

```diff
@@ -6,7 +6,7 @@ LATEST_TAG_WITH_V=`git describe --abbrev=0 --tags ${CIRCLE_SHA1}`
 LATEST_TAG=${LATEST_TAG_WITH_V:1}-approved
 
 # Send Slack notification
-../slack/notify_deploy.py $LATEST_TAG dev
+./scripts/slack/notify_deploy.py $LATEST_TAG dev
```

Comment:
This is because the command would be executed in the context in config.yml right?

Reply:
Yes :)